pkg/components/aws-ebs-csi-driver/component.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*Package awsebscsidriver - The golang plumbing for the aws-ebs-csi-driver
+component */
```

Comment:
It seems rather uncommon to me to see `/* */` style comments in Go. I'd use `//` instead.
```suggestion
// Package awsebscsidriver - The golang plumbing for the aws-ebs-csi-driver
// component.
```

Reply:
BTW, maybe we can create `doc.go` file as for other packages to keep this comment?

Reply:
Created a doc.go.